src/depcode.c

```diff
@@ -1,6 +1,7 @@
 #include "istore.h"
 #include "funcapi.h"
 #include "utils/array.h"
+#include "utils/builtins.h"
```

Comment:
that's already included in `istore.h`

Reply:
Leftovers from before the master was made 10.0 compatible. Will remove.